concepts/Common/TypeScript Guides/05 Scopes of Types/10 Component-Specific Types.md

```diff
@@ -0,0 +1,78 @@
+
+---
+##### jQuery
+
+Each DevExtreme component has its own set of types. To access these types, specify `DevExpress.ui.component` where `component` is the component name in camel case. Your IDE should give you a list of available types.
+
+    const dateType: DevExpress.ui.dxDateBox.DateType = 'datetime';
+
+##### Angular
+
+Each DevExtreme component has its own set of types. To access these types, import `DxComponentTypes` where `Component` is the component name.
+
+We export the `DxDateBoxTypes` [aggregated export](/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export) so you can import these types in your application:
```

Comment:
"We export... aggregated export" - this doesn't sound good. I'm not expert in the field, but even the technical part may not be right. 

How about this? 

```suggestion
For example, if you work with a `DateBox`, you need to import `DxDateBoxTypes`. This operation is possible because DevExtreme components use [aggregated export](/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export) for types.
```